src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -78,6 +78,19 @@ public string Title
         }
 
         private string _title = UtilsStrings.PromptForCredential_DefaultCaption;
+        
+        /// <summary>
+        /// Gets and sets whether to confirm the password.
+        /// </summary>
+        [Parameter(Mandatory = false)]
```

Comment:
It is default.
```suggestion
        [Parameter]
```

Reply:
Should I remove them from the other parameters as well? I only included it to align with the convention of the rest of the parameters, but I agree that it's not needed.

Reply:
No need fix other code without a request from maintainers. We prefer to separate style and functional changes by PRs/commits.